plugin/gf/user.vim

```diff
@@ -50,10 +50,11 @@ call s:define_ui_key_mappings()
 
 
 
-command! -bang -bar -nargs=0 GfUserDefaultKeyMappings
-\ call s:cmd_GfUserDefaultKeyMappings(<bang>0)
-function! s:cmd_GfUserDefaultKeyMappings(banged_p)
-  let modifier = a:banged_p ? '' : '<unique>'
+command! -bang -nargs=* GfUserDefaultKeyMappings
+\ call s:cmd_GfUserDefaultKeyMappings(<bang>0, <q-args>)
+function! s:cmd_GfUserDefaultKeyMappings(banged_p, args)
+  let modifier  = a:banged_p ? '' : '<unique>'
+  let modifier .= substitute(a:args, '\S\+', '<\0>', 'g')
```

Comment:
```suggestion
  let modifier .= a:args
```

I prefer just appending `a:args` as is.  It makes code using the command look more intuitive for users familiar with key mapping notation.

```vim
" Before
GfUserDefaultKeyMappings! buffer

" After
GfUserDefaultKeyMappings! <buffer>
```